### ISSUE TITLE
parameter values not updated when switching readers

### ISSUE THREAD

**Issue by jsmcibm** [primeqa/primeqa#418]
To reproduce:
My debugging deployment [link](http://9.59.197.117:84/reading) supports multiple readers.

The default reader `ExtractiveWithBooleanReader` has parameter `aggregate configuration` with value `/store/models/tydi_boolqa_config.json`.
When I switch to `ExtractiveReader`, the first parameter `Model` is shown with value `/store/models/tydi_boolqa_config.json`, the same value as the first parameter of the other reader.  
In other words the parameter names are updated to reflect the names of the new reader, but the parameter values are not updated.

![Screen Shot 2023-01-11 at 16 33 37](https://user-images.githubusercontent.com/102915688/211922549-bfeb5e7f-da32-4dd5-8ea1-f4a0bdab0ba7.png)
